core/src/main/resources/i18n/displayStrings_de.properties

```diff
@@ -52,6 +52,7 @@ shared.P2P=P2P
 shared.oneOffer=Angebot
 shared.multipleOffers=Angebote
 shared.Offer=Angebot
+shared.offerVolumeCode={0} Angebotsvolumen
```

Comment:
Any edit of the other language files will be reverted once updated from translators. For devs displayStrings.properties is the oinly file to edit, the rest is done via Transifex.

Reply:
Hi @chimp1984, 
Great! I will revert  the changes to the language files. Makes it easier !